components/raftstore-v2/src/operation/bucket.rs

```diff
@@ -26,71 +26,47 @@ use crate::{
 
 #[derive(Debug, Clone, Default)]
 pub struct BucketStatsInfo {
+    // the stats is increment flow.
     bucket_stat: Option<BucketStat>,
```

Comment:
The bucket_stat's write_bytes/write_qps/write_keys is accumulated by add_bucket_flow. 
When are these counters reset?